.github/workflows/merge-commits-to-next-branch.yml

```
@@ -29,72 +29,73 @@ jobs:
             const queueClient = queueServiceClient.getQueueClient(queueName);
             var receivedMessages = await queueClient.receiveMessages();
             const message = receivedMessages.receivedMessageItems[0];
-            $COMMIT_INFO = JSON.parse(message.messageText);
+            const firstMessage = JSON.parse(message.messageText);
+            return firstMessage;
+    outputs:
+      LABEL: ${{ fromJSON(steps.deque-message.outputs.result).label }}
+      AUTHOR: ${{ fromJSON(steps.deque-message.outputs.result).author }}
+      SHA: ${{ fromJSON(steps.deque-message.outputs.result).sha }}
   create-branch:
     runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-latest
    if: github.repository == 'microsoft/FluidFramework'
```

Adding this for all the jobs (or the top level ones) should prevent it from running outside the main repo.